docs/UserGuide.md

```diff
@@ -189,35 +193,45 @@ Clears all entries from QuickCache.
 
 Format: `clear`
 
-### Testing a flashcard : `test`
+### Testing a flashcard :
 
-Tests a specified flashcard from the list.
+#### Containing an open-ended question : `test INDEX ans/ANSWER`
 
-#### Containing an open-ended question
+<div markdown="block" class="alert alert-info">
+:bulb: Answer is <b>case insensitive</b>.
```

Comment:
I think this would better as an `:information_source:` ℹ️ emoji instead of a bulb